_includes/home/calendar.html

```diff
@@ -0,0 +1,110 @@
+<section class="calendar" id="calendar">
+
+<div class="container">
+    <div class="row">
```

Comment:
can you convert to 2-spaces for indents 🙏 ?

Reply:
fo sure